featuretools/computational_backends/utils.py

```diff
@@ -291,3 +292,43 @@ def replace_inf_values(feature_matrix, replacement_value=np.nan, columns=None):
     else:
         feature_matrix[columns] = feature_matrix[columns].replace([np.inf, -np.inf], replacement_value)
     return feature_matrix
+
+
+def get_ww_types_from_features(features, entityset, pass_columns, cutoff_time):
+    """ TODO """
```

Comment:
Should add a docstring here...or delete this. Ideally we would have a brief docstring though.